src/subcommand/sift_main.cpp

```diff
@@ -557,33 +556,27 @@ int main_sift(int argc, char** argv){
 
 
 
-if (alignment_file == "-"){
-    vg::io::for_each_interleaved_pair_parallel(cin, pair_filters);
-}
-else{
-    ifstream in;
-    in.open(alignment_file);
-    if (in.good()){
-
-        // if (just_calc_insert){
+    if (alignment_file == "-") {
+        vg::io::for_each_interleaved_pair_parallel(cin, pair_filters);
+    }
+    else {
+        ifstream in;
+        in.open(alignment_file);
+        // if (just_calc_insert) {
         //     vg::io::for_each_interleaved_pair_parallel(in, calc_insert);
         //     exit(0);
         // }
 
-        if (is_paired){
-            cerr << "Processing..." << endl;
+        if (is_paired) {
+            cerr << context << ": Processing..." << endl;
```

Comment:
Here this is touching essentially info-level logging lines, which is making it smell more and more like it's actually a logging framework.